cli/internal/core/engine.go

```diff
@@ -58,24 +67,6 @@ type EngineBuildingOptions struct {
 	TasksOnly bool
 }
 
-// Prepare constructs the Task Graph for a list of packages and tasks
-func (e *Engine) Prepare(options *EngineBuildingOptions) error {
-	pkgs := options.Packages
-	tasks := options.TaskNames
-	if len(tasks) == 0 {
-		// TODO(gsoltis): Is this behavior used?
-		for key := range e.Tasks {
-			tasks = append(tasks, key)
-		}
-	}
```

Comment:
This functionality is removed. As far as I can tell, this code would run _all_ known tasks if `turbo run` is run without any other arguments. But we're already catching this before this point, so this could should never be hit. cc @gsoltis let me know if you know any other history here.

Reply:
I think it was used in a test at one point, but if the tests are passing, then it's fine to nuke.